src/main/java/core/basesyntax/service/WriterToFile.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service;
+
+public interface WriterToFile {
+    void writeToFile(String toFileName, String report);
```

Comment:
Never forget about Abstraction. This interface can be used in many places in the future, it can write anything you want, not only reports.
```suggestion
    void writeToFile(String toFileName, String content);
```